pkg/kubectl/cmd/cp/cp.go

```diff
@@ -275,6 +284,24 @@ func (o *CopyOptions) copyToPod(src, dest fileSpec, options *exec.ExecOptions) e
 	return o.execute(options)
 }
 
+func (o *CopyOptions) checkTarExists(pod fileSpec) error {
```

Comment:
Does this actually check the difference between "`which` not found", and "`tar` not found"?

Reply:
I removed which dependency